release/action.yaml

```diff
@@ -34,10 +34,13 @@ runs:
   steps:
     - name: Checkout
       uses: actions/checkout@v3
-      if: inputs.checkout-repo
```

Comment:
Why is this guard clause being removed? Looks like this should instead be `if: inputs.checkout-repo == 'true'` and that particular guard clause is not consistent throughout `open-turo`, which I will address.

Reply:
This is removed because it's now used in the line before. I'm on m phone but see the latest merged pr for more context

Reply:
This guard is just meant to be used for the force depth setting in the line below